src/Camera.jl

```diff
@@ -28,19 +28,36 @@ export serial, model, vendor, isrunning, start!, stop!, getimage, getimage!, sav
 """
 mutable struct Camera
   handle::spinCamera
+  names::Dict{String, String}
 
   function Camera(handle)
     @assert spinsys.handle != C_NULL
     @assert handle != C_NULL
     spinCameraDeInit(handle)
     spinCameraInit(handle)
-    cam = new(handle)
+    names = Dict{String, String}()
+    cam = new(handle, names)
     finalizer(_release!, cam)
+    
     # Activate chunk mode
     set!(SpinBooleanNode(cam, "ChunkModeActive"), true)
     _chunkselect(cam, ["FrameID", "FrameCounter"], "frame indentification")
     _chunkselect(cam, ["ExposureTime"], "exposure time")
     _chunkselect(cam, ["Timestamp"], "timestamps")
+
+    # Discover ambiguous names
+    cam.names["AutoExposureTimeLowerLimit"] = "AutoExposureTimeLowerLimit"
+    cam.names["AutoExposureTimeUpperLimit"] = "AutoExposureTimeUpperLimit"
+    
+    try
+      Spinnaker.get(Spinnaker.SpinIntegerNode(cam, "AutoExposureTimeLowerLimit"))
+    catch
+      @info "renameing exposure"
```

Comment:
I'd go for something a bit clearer:
`@info "Renaming AutoExposureTimeLowerLimit & AutoExposureTimeUpperLimit"`